lib/java/src/test/java/org/apache/thrift/server/ServerTestBase.java

```diff
@@ -246,7 +246,7 @@ public Map<Long, Map<Numberz, Insanity>> testInsanity(Insanity argument) {
       first_map.put(Numberz.TWO, argument);
       first_map.put(Numberz.THREE, argument);
 
-      Insanity looney = new Insanity();
+      Insanity looney = new Insanity(new HashMap<>(), Arrays.asList());
```

Comment:
would this break if not added?

Reply:
Yes, since this field is marked as non-optional in the TestThrift.thrift spec, the Swift generator makes them required fields in Swift for deserialization.  I'm not sure if this is the intended behaviour, maybe "non-optional" fields should be mapped to optionals in Swift since they can apparently be omitted by servers according to the spec for "default requiredness".  We could just comment out the insanity test for now until changes are made to the Swift generator to be more compliant with the spec